nixos/modules/services/misc/gitolite.nix

```diff
@@ -143,21 +143,37 @@ in
     users.users.${cfg.user} = {
       description     = "Gitolite user";
       home            = cfg.dataDir;
-      createHome      = true;
       uid             = config.ids.uids.gitolite;
       group           = cfg.group;
       useDefaultShell = true;
     };
     users.groups."${cfg.group}".gid = config.ids.gids.gitolite;
 
+    systemd.tmpfiles.rules = [
+      "d '${cfg.dataDir}' 0750 ${cfg.user} ${cfg.group} - -"
+      "d '${cfg.dataDir}'/.gitolite - ${cfg.user} ${cfg.group} - -"
+      "d '${cfg.dataDir}'/.gitolite/logs - ${cfg.user} ${cfg.group} - -"
```

Comment:
why using hidden files there ?

Reply:
It's the default directory, mentioned in [Gitolites documentation](https://gitolite.com/gitolite/files.html#directories).

Reply:
ok thanks. I would have liked to try this on my server but a bit tight on time. Looks good so let's merge.